msg/templates/uorb/msg.h.template

```diff
@@ -83,7 +84,7 @@ for field in spec.parsed_fields():
 @# Constants c style
 #ifndef __cplusplus
 @[for constant in spec.constants]@
-#define @(constant.name) @(int(constant.val))
+#define @(uorb_struct_upper)_@(constant.name) @(int(constant.val))
```

Comment:
Question of taste: why not use spec.short_name.upper()  ? uorb_struct has an additional _s

Reply:
Some vague sense of consistency. It's topic_s for the generated header and topic for the ORB_ID declaration.

So throughout the codebase we'd have `topic_s::CONSTANT` in c++ and now `TOPIC_S_CONSTANT` in C.

Reply:
I could go either way.

Reply:
Can we trim the "s" please? I think outside of a struct name it makes little sense (and I regret it in the struct name anyway).

Reply:
Done